### PR TITLE
New_PR

### PR DESCRIPTION
**Pull request by CeO-PsH**

### CODE REVIEW

File: py-count-occurrences
Comment (Dimosphen1) [inline py-count-occurrences:1]: Remove this submodule from the commit.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]

File: py-count-occurrences
Comment (CeO-PsH) [reply py-count-occurrences:1]: Can you please clarify what needs to be done?

Comment (Polyakiv-Andrey) [reply py-count-occurrences:1]: You need to write in terminal next command 
git rm <file name>

Comment (Polyakiv-Andrey) [reply py-count-occurrences:1]: You need to write in terminal next command
git rm file_name

**Pull request closed without merging**